### PR TITLE
Update airflow.md

### PR DESCRIPTION
**Pull request by isichei**
Adding in documentation for new operator

### CODE REVIEW

File: source/documentation/airflow.md [hunk 1]
```diff
@@ -207,6 +251,9 @@ dag = DAG(
     schedule_interval=None,
 )
 
+# It is good practice to use the same name for teh task_id and task_name.
```
Comment (AnthonyCody) [inline source/documentation/airflow.md:254]: Typo: teh

Comment (AnthonyCody) [reply source/documentation/airflow.md:254]: Also:  `task_name` or `name` ?

Comment (isichei) [reply source/documentation/airflow.md]: Have fixed and clarified

Comment (AnthonyCody) [reply source/documentation/airflow.md]: OK, should line 267 be:
`name=task_id`
or
`task_name=task_id`

**Pull request merged**